livestyled/schemas/booking.py

```diff
@@ -0,0 +1,20 @@
+from marshmallow import EXCLUDE, fields, Schema
+
+from livestyled.models.booking import Booking
+
+
+class BookingSchema(Schema):
+    class Meta:
+        unknown = EXCLUDE
+        model = Booking
+        api_type = 'bookings'
+        url = 'v4/bookings'
+
+    id = fields.Int()
+    title = fields.String()
+    device = fields.String()
+    user = fields.String()
+    event = fields.String()
+    created_at = fields.String()
+    updated_at = fields.String()
```

Comment:
you need to change these 2 to:

`created_at = fields.String(data_key='createdAt')`
`updated_at = fields.String(data_key='updatedAt')`

`data_key` is the key it'll look for in the json it is deserializing from the API response and also what it'll output when serializing to send back to the API.